sormas-ui/src/main/java/de/symeda/sormas/ui/caze/eventLink/EventListComponent.java

```diff
@@ -37,14 +43,29 @@
 import de.symeda.sormas.ui.UserProvider;
 import de.symeda.sormas.ui.utils.CssStyles;
 
+import java.util.List;
+
 public class EventListComponent extends VerticalLayout {
 
 	private EventList list;
 	private Button createButton;
 
 	public EventListComponent(CaseReferenceDto caseRef) {
 
-		createEventListComponent(new EventList(caseRef), e -> ControllerProvider.getEventController().selectOrCreateEvent(caseRef));
+		createEventListComponent(new EventList(caseRef), e -> {
+
+			//check if there are active events in the database
+			List<String> events = FacadeProvider.getEventFacade().getAllActiveUuids();
```

Comment:
Please use a "count" method instead for performance reasons.

Reply:
ok will do